app/renderer/lib/client-frameworks/ruby.js

```diff
@@ -298,29 +337,33 @@ driver.quit`;
   }
 
   codeFor_getUrl () {
-    return `current_url = driver.current_url`;
+    return `current_url = driver.get_current_url`;
+  }
+
+  codeFor_back () {
+    return `driver.back`;
   }
 
   codeFor_forward () {
-    return `driver.navigate().forward`;
+    return `driver.forward`;
   }
 
   codeFor_refresh () {
-    return `driver.navigate().refresh`;
+    return `driver.refresh`;
   }
 
   // Context
 
   codeFor_getContext () {
-    return `driver.current_context`;
+    return `context = driver.current_context`;
   }
 
   codeFor_getContexts () {
-    return `driver.available_contexts`;
+    return `contexts = driver.available_contexts`;
   }
 
   codeFor_switchContext (varNameIgnore, varIndexIgnore, name) {
-    return `driver.context = '${name}'`;
+    return `driver.set_context '${name}'`;
```

Comment:
Both work as https://github.com/appium/ruby_lib_core/blob/1d438743e7b88227dfe0d1477955da61377cb401/lib/appium_lib_core/common/base/driver.rb#L472-L473
```suggestion
    return `driver.context = '${name}'`;
```

`driver.context =` is more Ruby friendly syntax, so here can keep the previous one